doc/conf_ref.rst

```diff
@@ -152,6 +152,22 @@ one main section. The repository sections define the configuration for each
 
     If enabled, DNF will continue running and disable the repository that couldn't be contacted for any reason when downloading metadata. This option doesn't affect skipping of unavailable packages after dependency resolution. To check inaccessibility of repository use it in combination with :ref:`refresh command line option <refresh_command-label>`. The default is True.
 
+================
+ Repo Variables
+================
+
+``$releasever``
+
+    Refers to the release version of operating system detected by RPM.
```

Comment:
Let's not blame RPM. DNF is the one who does the magic. I'd rather say that it's derived from the information available in RPMDB (like here: http://dnf.readthedocs.org/en/latest/api_rpm.html#dnf.rpm.detect_releasever) or that it's obtained from "redhat-release" package (like in the System Administrator's Guide; but that is not completely true).